model/src/service/util.py

```diff
@@ -45,6 +45,7 @@ def create_point(username, measurement, value, time):
     }
 
     return json_body
+
```

Comment:
We do have some white space changes which are not required, it makes me thinking of some code-style as pre-commit actions like pep-8 or ...

Reply:
sounds interesting, I haven't worked with that before but we can discuss how to use it :)